config/locales/en.yml

```
@@ -241,3 +241,12 @@ en:
               - "I cannot leave home because I think I’m at risk of severe illness from coronavirus"
               - "I cannot go out because I have a disability"
               - "I’m unable to leave my home for another reason"
+    results:
+      header:
+        context: Coronavirus (COVID-19)
+        title: Information based on your answers
+        start_again_text: Start again
+      feedback:
+        text: Help us improve
+        link_text: Give feedback on this service
+        link_href: "https://www.gov.uk/done/coronavirus-support-from-business"
```

This should be updated
```suggestion
        link_href: "https://www.gov.uk/done//find-coronavirus-support"
```

---

great spot, resolved.